modules/client/src/validation/addresses.ts

```diff
@@ -15,15 +15,15 @@ function isValidAddress(value: any): boolean {
 
 export function invalidXpub(value: string): string | undefined {
```

Comment:
in all of these functions we should probably allow for more specific errors to be thrown, ie add a way to optionally append or prepend more contextual messages to the errors

Reply:
Right, that's a good idea. Separate from the scope of this PR though.